tools/e2etesting/DeployEdge.ps1

```diff
@@ -71,13 +71,8 @@ if ($iotHub.Count -ne 1) {
 Write-Host "IoT Hub Name: $($iotHub.Name)"
 
 ## Check if KeyVault exists
-$keyVault = Get-AzKeyVault -ResourceGroupName $ResourceGroupName
-
-if ($keyVault.Count -ne 1) {
-    Write-Error "keyVault could not be automatically selected in Resource Group '$($ResourceGroupName)'."    
-} 
-
-Write-Host "Key Vault Name: $($keyVault.VaultName)"
+$keyVault = "e2etestingkeyVault" + $testSuffix
```

Comment:
do we know for sure the Key Vault already exists?

Reply:
yes, it's created in one of the first steps

Reply:
the reason I removed the check is that this check sometimes fail even though the KeyVault exists. I think it's an error in this cmdlet